mypy/test/teststubtest.py

```diff
@@ -1068,6 +1068,26 @@ def spam(x=Flags4(0)): pass
             """,
             error="spam",
         )
+        yield Case(
+            stub="""
+            from typing_extensions import Final, Literal
+            class BytesEnum(bytes, enum.Enum):
+                a: bytes
+            FOO: Literal[BytesEnum.a]
```

Comment:
It's somewhat unlikely to come up in practice, but stubtest emits a false positive on this line using mypy `master`